server/src/main/java/org/cloudfoundry/identity/uaa/client/VmidentityQueryableClientDetailsService.java

```diff
@@ -34,8 +34,7 @@ protected VmidentityQueryableClientDetailsService(MultitenantVmidentityClientDet
 
     @Override
     public List<ClientDetails> query(String filter, String sortBy, boolean ascending) {
```

Comment:
We'll fix that up with the task to move the approvals into IDM (#17) - currently everything is still wired through the `JdbcApprovalStore`